crates/uv-build-backend/src/metadata.rs

```diff
@@ -530,68 +534,68 @@ impl PyProjectToml {
-    license: Option<License>,
+    pub(crate) license: Option<License>,
     /// The paths to files containing licenses and other legal notices to be distributed with the
     /// project.
     ///
     /// From the provisional PEP 639
-    license_files: Option<Vec<String>>,
+    pub(crate) license_files: Option<Vec<String>>,
     /// The people or organizations considered to be the "authors" of the project.
-    authors: Option<Vec<Contact>>,
+    pub(crate) authors: Option<Vec<Contact>>,
     /// The people or organizations considered to be the "maintainers" of the project.
-    maintainers: Option<Vec<Contact>>,
+    pub(crate) maintainers: Option<Vec<Contact>>,
     /// The keywords for the project.
-    keywords: Option<Vec<String>>,
+    pub(crate) keywords: Option<Vec<String>>,
     /// Trove classifiers which apply to the project.
-    classifiers: Option<Vec<String>>,
+    pub(crate) classifiers: Option<Vec<String>>,
     /// A table of URLs where the key is the URL label and the value is the URL itself.
     ///
     /// PyPI shows all URLs with their name. For some known patterns, they add favicons.
     /// main: <https://github.com/pypi/warehouse/blob/main/warehouse/templates/packaging/detail.html>
     /// archived: <https://github.com/pypi/warehouse/blob/e3bd3c3805ff47fff32b67a899c1ce11c16f3c31/warehouse/templates/packaging/detail.html>
-    urls: Option<BTreeMap<String, String>>,
+    pub(crate) urls: Option<BTreeMap<String, String>>,
     /// The console entrypoints of the project.
     ///
     /// The key of the table is the name of the entry point and the value is the object reference.
-    scripts: Option<BTreeMap<String, String>>,
+    pub(crate) scripts: Option<BTreeMap<String, String>>,
     /// The GUI entrypoints of the project.
     ///
     /// The key of the table is the name of the entry point and the value is the object reference.
-    gui_scripts: Option<BTreeMap<String, String>>,
+    pub(crate) gui_scripts: Option<BTreeMap<String, String>>,
     /// Entrypoints groups of the project.
     ///
     /// The key of the table is the name of the entry point and the value is the object reference.
-    entry_points: Option<BTreeMap<String, BTreeMap<String, String>>>,
+    pub(crate) entry_points: Option<BTreeMap<String, BTreeMap<String, String>>>,
     /// The dependencies of the project.
-    dependencies: Option<Vec<Requirement>>,
+    pub(crate) dependencies: Option<Vec<Requirement>>,
     /// The optional dependencies of the project.
-    optional_dependencies: Option<BTreeMap<ExtraName, Vec<Requirement>>>,
+    pub(crate) optional_dependencies: Option<BTreeMap<ExtraName, Vec<Requirement>>>,
     /// Specifies which fields listed by PEP 621 were intentionally unspecified so another tool
     /// can/will provide such metadata dynamically.
     ///
     /// Not supported, an error if anything but the default empty list.
-    dynamic: Option<Vec<String>>,
+    pub(crate) dynamic: Option<Vec<String>>,
```

Comment:
Do you need access to everything? Would it be reasonable to expose only what you need through methods?

(I suggest this for abstract "encapsulation is good" reasons, and nothing in particular that is concrete. And I do sympathize with just making internals crate-public as that can often be easier.)